README.old.md

```diff
@@ -0,0 +1,2 @@
+# to-do-using-react-js
```

Comment:
[OPTIONAL] it might be a good idea to delete this file, just so as to keep the repo cleaner